appthemehelper/build.gradle

```diff
@@ -1,12 +1,12 @@
 apply plugin: 'com.android.library'
 apply plugin: 'kotlin-android'
 android {
-    compileSdk 35
+    compileSdk 34
```

Comment:
Why are you downgrading the sdk?

Reply:
> Why are you downgrading the sdk?

I had to because one file gave me many null-safety errors, so I came up with the simple solution :D (I'll try to find specifically which file was that)

Reply:
Just realized I forgot to upgrade the app module to targetSdk 35, so I guess this is fine for now

Reply:
> Just realized I forgot to upgrade the app module to targetSdk 35, so I guess this is fine for now

Solved that problem as well ✅